README.md

```diff
@@ -85,12 +85,25 @@ Used Redux extensions:
 You can run the linter by calling **yarn run lint**. Some issues can be fixed automatically by running **yarn run lintfix**
 
 ## IDE
-Just import this project (from existing sources). Run **yarn** and right-click on package.json to show the npm scripts. 
-Also mark the *src* and *public* folder as *Source directory*. In IntelliJ no other plugins are needed. 
+**Make sure you have at least [nodejs 6](https://nodejs.org/) installed**
+
+We suggest IntellJ IDEA Ultimate as IDE. Just import this project (from existing sources).
+Run **yarn** in Terminal and right-click on package.json to show the npm scripts. 
+Also mark the *src* and *public* folder as *Source directory*.
```

Comment:
i think we renamed the "public" to "www", and at least in intelliJ version 2017.3 the option is called mark as "Sources Root'

Reply:
done